modules/api/php/module.class.inc

```diff
@@ -114,4 +129,14 @@ class Module extends \Module
     {
         return "API";
     }
+
+    /**
+     * This module have a specification
+     *
+     * @return bool Always true
+     */
+    public function hasAPISpecification() : bool
```

Comment:
I'm not crazy about this being a bool instead of either an array of files or URLs, so that modules could include multiple specifications if they wanted (ie. for the different API versions), but I'm not going to block over it either. It can always be changed later if need be..

Reply:
I don't mind having it return an array of URI 
I'll just need to rework https://github.com/aces/Loris/pull/5354/files#diff-9474e3d52be30a49c24701492f60ce04de4404ab90f0a617aaa58f27e6dc37f9R74

Reply:
@driusan are you still okai with a followup PR for this ?

Reply:
I'm okay with it being fixed as a followup